### ISSUE TITLE
Pacific/Kanton timezone not supported

### ISSUE THREAD

**Issue by derekantrican** [mattjohnsonpint/TimeZoneConverter#106]
I had a user of my program on a M1 Mac get a `TimeZoneNotFoundException` when they hit the following code (id hardcoded here for ease of repro): `TZConvert.GetTimeZoneInfo("Pacific/Kanton")` . Not sure if this timezone was recently added and is not yet supported by this library or what.

I'm aware you're trying to stop maintaining this project, but logging here for visibility

**Comment by jnyrup** [comment]
> Rename Pacific/Enderbury to Pacific/Kanton.

[Release notes of tzdb 2021b](https://github.com/eggert/tz/blob/228a93f759cd5f2b0cca9e05eeee3d7cdea07ebf/NEWS#L94)

Here's the results from my rebuilt of TimeZoneConverter against NodaTime 2.4.14, which includes the changes from tzdb 2021b
```cs
var kanton = TimeZoneConverter.TZConvert.GetTimeZoneInfo("Pacific/Kanton");
var enderbury = TimeZoneConverter.TZConvert.GetTimeZoneInfo("Pacific/Enderbury");
var same = ReferenceEquals(kanton, enderbury); // true
```

**Comment by derekantrican** [comment]
Thanks for looking into this @jnyrup ! Glad to see the release notes confirming the change.

I came across this due to code that is essentially:
```csharp
TimeZoneInfo.GetSystemTimeZones().Select(t => TZConvert.GetTimeZoneInfo(t.Id));
```

Maybe this could be converted into a unit test and run nightly with something like Travis CI or GitHub Actions?

**Comment by jnyrup** [comment]
This is essentially what the existing unit test `TimeZoneConverter.Tests.IanaToWindowsTests.Can_Convert_IANA_System_Zones_To_Windows()` does.

**Comment by mattjohnsonpint** [comment]
> I'm aware you're trying to stop maintaining this project, but logging here for visibility

Nope.  I'm not abandoning this project. I'm just tied up with other things at the moment.  I'll have an update in the next week or so.  Thanks.

**Comment by mattjohnsonpint** [comment]
Fixed with version 4.0.0.  Thanks for your patience.
